dawn/src/dawn/Optimizer/PassComputeStageExtents.cpp

```diff
@@ -36,6 +37,12 @@ bool PassComputeStageExtents::run(
     // backward loop over stages
     for(int i = numStages - 1; i >= 0; --i) {
       iir::Stage& fromStage = *(stencil.getStage(i));
+      for(auto globalIterationSpace : fromStage.getIterationSpace()) {
+        if(globalIterationSpace) {
+          fromStage.setExtents(iir::Extents());
+          continue;
```

Comment:
not sure what this statement is supposed to do, but certainly it does not what it should as it will only skip the rest of the current iteration (which is nothing).

Reply:
you are right, fixed